src/docs/arch-eviction.dox

```diff
@@ -0,0 +1,14 @@
+/*! @arch_page arch-eviction Eviction
+
+Eviction represents the process of writing dirty data to disk.
```

Comment:
Doesn't eviction cover include reclaiming space from clean pages, as well?

Reply:
Yes, added that.